doc/src/running-suites.rst

```diff
@@ -0,0 +1,1634 @@
+priority jobs. The preempted jobs may then be automatically restarted
+by the resource manager, from the same point (if suspended) or requeued
+to run again from the start (if killed).
+
+Suspended jobs will poll as still running (their job status file says they
+started running, and they still appear in the resource manager queue).
+Loadleveler jobs that are preempted by kill-and-requeue ("job vacation") are
+automatically returned to the submitted state by Cylc.  This is possible
+because Loadleveler sends the SIGUSR1 signal before SIGKILL for preemption.
+Other batch schedulers just send SIGTERM before SIGKILL as normal, so Cylc
+cannot distinguish a preemption job kill from a normal job kill. After this the
+job will poll as failed (correctly, because it was killed, and the job status
+file records that). To handle this kind of preemption automatically you could
+use a task failed or retry event handler that queries the batch scheduler queue
+(after an appropriate delay if necessary) and then, if the job has been
+requeued, uses ``cylc reset`` to reset the task to the submitted state.
+
+
+Manual Task Triggering and Edit-Run
+-----------------------------------
+
+Any task proxy currently present in the suite can be manually triggered at any
+time using the ``cylc trigger`` command, or from the right-click task
+menu in gcylc. If the task belongs to a limited internal queue
+(see :ref:`InternalQueues`), this will queue it; if not, or if it is already
+queued, it will submit immediately.
+
+With ``cylc trigger --edit`` (also in the gcylc right-click task menu)
+you can edit the generated task job script to make one-off changes before the
+task submits.
+
+
+.. _cylc-broadcast:
+
+Cylc Broadcast
+--------------
+
+The ``cylc broadcast`` command overrides ``[runtime]``
+settings in a running suite. This can
+be used to communicate information to downstream tasks by broadcasting
+environment variables (communication of information from one task to
+another normally takes place via the filesystem, i.e. the input/output
+file relationships embodied in inter-task dependencies). Variables (and
+any other runtime settings) may be broadcast to all subsequent tasks,
+or targeted specifically at a specific task, all subsequent tasks with a
+given name, or all tasks with a given cycle point; see broadcast command help
+for details.
+
+Broadcast settings targeted at a specific task ID or cycle point expire and
+are forgotten as the suite moves on. Un-targeted variables and those
+targeted at a task name persist throughout the suite run, even across
+restarts, unless manually cleared using the broadcast command - and so
+should be used sparingly.
+
+
+The Meaning And Use Of Initial Cycle Point
+------------------------------------------
+
+When a suite is started with the ``cylc run`` command (cold or
+warm start) the cycle point at which it starts can be given on the command
+line or hardwired into the suite.rc file:
+
+.. code-block:: cylc
```

Comment:
This code block renders in red as an error but this time it's actually a genuine error!

```diff
-.. code-block:: cylc
+.. code-block:: bash
```

There are a few other code blocks showing up in red (most are probably my fault, I'll fix the syntax file) some are genuine. Here is a list of the effected files:

```console
$ grep -l --color=none -r '<span class="err"' | sed 's/\.html/\.rst/' | sed 's/^/doc\/src\//'
doc/src/install/html/built-sphinx/appendices/cylc-6-migration-ref.rst
doc/src/install/html/built-sphinx/appendices/suiterc-config-ref.rst
doc/src/install/html/built-sphinx/running-suites.rst
doc/src/install/html/built-sphinx/suite-config.rst
doc/src/install/html/built-sphinx/suite-design-guide/general-principles.rst
doc/src/install/html/built-sphinx/suite-design-guide/portable-suites.rst
doc/src/install/html/built-sphinx/suite-design-guide/style-guide.rst
doc/src/built-sphinx/appendices/cylc-6-migration-ref.rst
doc/src/built-sphinx/appendices/suiterc-config-ref.rst
doc/src/built-sphinx/running-suites.rst
doc/src/built-sphinx/suite-config.rst
doc/src/built-sphinx/suite-design-guide/general-principles.rst
doc/src/built-sphinx/suite-design-guide/portable-suites.rst
doc/src/built-sphinx/suite-design-guide/style-guide.rst
```

Reply:
Nice, thanks for extracting that as a list for me. I'll get those errors ironed out.